src/main/java/org/lsc/plugins/connectors/james/JamesClientException.java

```diff
@@ -0,0 +1,32 @@
+package org.lsc.plugins.connectors.james;
```

Comment:
license?

Reply:
I had a question regarding the license.

I saw Tung mentionning you in the licenses. Shouldn't he put his name instead there in the files he created?

Reply:
that's what I suggested https://github.com/lsc-project/lsc-james-plugin/pull/1#discussion_r651523679

he can also keep both if he thinks his work is a derived work of mine, but for my part I don't care